isis/scripts/downloadIsisData

```diff
@@ -110,7 +113,7 @@ def create_rclone_arguments(destination, mission_name, ntransfers=10, rclone_kwa
     if source_type == "naifKernels":
         destination = os.path.join(destination, "kernels")
 
-    extra_args=[f"{mission_name}",f"{destination}", "--progress", f"--checkers={ntransfers}", f"--transfers={ntransfers}", "--track-renames", f"--log-level={log.getLevelName(log.getLogger().getEffectiveLevel())}"]
+    extra_args=[f"{mission_name}",f"{destination}", "--progress", f"--checkers={ntransfers}", f"--transfers={ntransfers}", "--track-renames", f"--log-level={log.getLevelName(log.getLogger().getEffectiveLevel())}", f"--exclude={exclude_string}"]
```

Comment:
You might want to check for additional --include or --filter flags passed into by the user and merge them since rclone docs say mixing them can cause weird behavior.